### DISCUSSION TITLE
Clarity needed on github slug

### DISCUSSION THREAD

**Discussion by mattjared** [avitorio/outstatic#46] [Ideas]
Might just be me but it took a few tries and cracking opening the console to look at error messages to get exactly what the github slug is. Might be worth it to add a comment referencing just the project name is what OST is looking for in that env variable. Just a thought, thanks again for the awesome work on this CMS! 🚀

**Comment by avitorio** [comment]
Hi @mattjared,

I'm glad you got it working and I'm sorry for the confusion with the env variable.

Did you follow the docs? I think I'll edit the env variables part to use bullet points or a table instead of giving an actual .env file example. If you have suggestions on wording or anything else, let me know.

Thanks for sharing your experience.
